projects/plugins/jetpack/tests/php/sync/test_class.jetpack-sync-options.php

```diff
@@ -176,6 +176,7 @@ public function test_sync_default_options() {
 			'jetpack_mailchimp'                            => '{}',
 			'jetpack_protect_key'                          => 'pineapple',
 			'jetpack_protect_global_whitelist'             => 'pineapple',
+			'jetpack_waf_automatic_rules'                  => 'pineapple',
```

Comment:
I wasn't 100% certain how to run this specific unit test manually to verify they pass, but I assume they run as part of the PR checks?